### PR TITLE
P4 1665 move profile controller under v1

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link

https://dsdmoj.atlassian.net/browse/P4-1665

### What?
This PR move the /v2/profile_controllers into /V1/profile_controllers

### Why?
Creating a new API version is not necessary/premature,
so we are stepping back adding this functionality in V1

- [X] Updated API docs if necessary

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -23,6 +23,8 @@
         get 'images', to: 'people#image'
         get 'court_cases', to: 'people#court_cases'
         get 'timetable', to: 'people#timetable'
+
+        resources :profiles, only: %i[create update]
```
[the remark this answers is not in the document]
Comment (alexdesi) [reply config/routes.rb:27]: It was in the controller API, I guess it's good to have an endpoint to patch the profile. but not sure if there is FE scenario that uses it atm.

**Comment by alexdesi** [comment]
Now, the profile has only one serializer, the v2, that one without `first_names, last_names,`.

File: app/serializers/move_serializer.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ class MoveSerializer < ActiveModel::Serializer
              :date_to
 
   has_one :person, serializer: PersonSerializer
-  has_one :profile, serializer: ProfileSerializer
+  has_one :profile, serializer: ProfileSerializer # <- TODO: update the serializer
```
Comment (willfish) [inline app/serializers/move_serializer.rb:22]:
```suggestion
  has_one :profile, serializer: ProfileSerializer
```

File: app/serializers/profile_serializer.rb [hunk 3]
```diff
@@ -2,25 +2,10 @@
...
-  SUPPORTED_RELATIONSHIPS = %i[ethnicity gender].freeze
-
-  def assessment_answers
-    object.assessment_answers || []
-  end
-
-  def identifiers
-    object.profile_identifiers || []
-  end
+  SUPPORTED_RELATIONSHIPS = %w[person].freeze
```
Comment (willfish) [inline app/serializers/profile_serializer.rb:10]: 👍

File: spec/serializers/profile_serializer_spec.rb [hunk 4]
```diff
@@ -17,18 +17,9 @@
     expect(result[:data][:id]).to eql profile.id
   end
 
-  it 'contains first_names attribute' do
```
Comment (willfish) [inline spec/serializers/profile_serializer_spec.rb:20]: 👍

File: spec/serializers/profile_serializer_spec.rb [hunk 5]
```diff
@@ -17,18 +17,9 @@
...
-  end
-
-  it 'contains last_name attribute' do
-    expect(result[:data][:attributes][:last_name]).to eql profile.last_name
-  end
-
   describe '#assessment_answers' do
     let(:risk_alert_type) { create :assessment_question, :risk }
     let(:health_alert_type) { create :assessment_question, :health }
-    let(:court_type) { create :assessment_question, :court }
```
Comment (willfish) [inline spec/serializers/profile_serializer_spec.rb:31]: Curious about this one

File: spec/serializers/profile_serializer_spec.rb [hunk 6]
```diff
@@ -46,103 +37,20 @@
...
-            description: gender&.description,
-          },
-        },
-      ]
-    end
-
-    it 'contains an included gender' do
-      expect(result[:included]).to(include_json(expected_json))
-    end
+    context 'with empty assessment_answers' do
```
Comment (willfish) [inline spec/serializers/profile_serializer_spec.rb:48]: Given this is what a profile is, I'm not sure whether this is even a valid situation 🤔

Comment (alexdesi) [reply spec/serializers/profile_serializer_spec.rb:48]: true, maybe we should add a validation for that, to not allow empty assessment_answers

**Pull request merged**